tests/fakeapi/example_method_view.py

```diff
@@ -6,19 +6,22 @@ class PetsView(MethodView):
     mycontent = "demonstrate return from MethodView class"
 
     def get(self, **kwargs):
-        kwargs.update({"method": "get"})
```

Comment:
These responses were not conforming to the spec, but were not checked correctly before.